extensions/amp-carousel/0.1/slidescroll.js

```diff
@@ -263,12 +263,12 @@ export class AmpSlideScroll extends BaseSlides {
   /** @override */
   onLayoutMeasure() {
     this.slideWidth_ = this.getLayoutWidth();
-    if (this.slideIndex_ !== null) {
+    if (!this.hasNativeSnapPoints_ && this.slideIndex_ !== null) {
```

Comment:
tested in iOS? worried this may break something in there that for whatever reason needs it.

Reply:
Yup tested in iOS using the iPhone X simulator available on XCode.

Reply:
👍 short circuit `this.hasNativeSnapPoints_` above `getLayoutWidth` ?